internal/operators/lvm/lvm_operator.go

```diff
@@ -0,0 +1,190 @@
+	}
+	minOpenshiftVersionForLvm, err = version.NewVersion(o.config.LvmMinOpenshiftVersion)
+	if err != nil {
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{err.Error()}}, nil
+	}
+	if ocpVersion.LessThan(minOpenshiftVersionForLvm) {
+		message := "ODF LVM operator is only supported for openshift versions 4.11.0 and above"
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetClusterValidationID(), Reasons: []string{message}}, nil
+	}
+
+	return api.ValidationResult{Status: api.Success, ValidationId: o.GetClusterValidationID()}, nil
+}
+
+// ValidateHost always return "valid" result
+func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, host *models.Host) (api.ValidationResult, error) {
+	if host.Inventory == "" {
+		message := "Missing Inventory in the host"
+		return api.ValidationResult{Status: api.Pending, ValidationId: o.GetHostValidationID(), Reasons: []string{message}}, nil
+	}
+
+	inventory, err := common.UnmarshalInventory(host.Inventory)
+	if err != nil {
+		message := "Failed to get inventory from host"
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{message}}, err
+	}
+
+	// GetValidDiskCount counts the total number of valid disks in each host and return an error if we don't have the disk of required size
+	diskCount, err := o.getValidDiskCount(inventory.Disks, host.InstallationDiskID)
+	if err != nil {
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{err.Error()}}, nil
+	}
+	if diskCount == 0 {
+		message := "Insufficient disks, ODF LVM requires at least one non-installation disk on the host"
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{message}}, nil
+	}
+
+	requirements, err := o.GetHostRequirements(ctx, cluster, host)
+	if err != nil {
+		message := fmt.Sprintf("Failed to get the host requirements for host with id %s", host.ID)
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{message, err.Error()}}, err
+	}
+
+	cpu := requirements.CPUCores
+	if inventory.CPU.Count < cpu {
+		message := fmt.Sprintf("Insufficient CPU to deploy ODF LVM. The required CPU count is %d but found %d", cpu, inventory.CPU.Count)
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{message}}, nil
+	}
+
+	memory := requirements.RAMMib
+	memoryBytes := conversions.MibToBytes(memory)
+	if inventory.Memory.UsableBytes < memoryBytes {
+		usableMemory := conversions.BytesToMib(inventory.Memory.UsableBytes)
+		message := fmt.Sprintf("Insufficient memory to deploy ODF LVM. The required memory is %d MiB but found %d MiB", memory, usableMemory)
+		return api.ValidationResult{Status: api.Failure, ValidationId: o.GetHostValidationID(), Reasons: []string{message}}, nil
+	}
```

Comment:
If I understand correctly, this compares the host's CPU and RAM to the amount needed to run the operator.  Shouldn't it compare the host's CPU and RAM to the amount needed to run OpenShift plus all enabled operators?